### ISSUE TITLE
Sites might have a solid guess about user identity

### ISSUE THREAD

**Issue by martinthomson** [fedidcg/FedCM#322]
https://fedidcg.github.io/FedCM/#browser-surface-impersonation says:

> This is not possible because the FedCM UI uses the metadata about the user accounts of the [IDP](https://fedidcg.github.io/FedCM/#idp), which the website doesn’t have access.

Aside from the grammar problem here, this is probably not a good basis for a design.  Presenting this as an absolute, as this does, might be unwise.  A site might be able to make a good guess about user identity.  Therefore, if the goal of an attacker is to confirm a guess about identity, any design that relies on this property might not be sufficient.

Being able to confirm (or re-confirm) identity linkages is possibly less useful than invoking the API for real, but this can at least be accurate.

**Comment by npm1** [comment]
Isn't the claim explain the sentence right after? `If this is a malicious site, it would not know the user accounts, otherwise the user is already compromised.` Anyways, we can reword this to make it less absolute.
